collect_app/src/main/AndroidManifest.xml

```diff
@@ -40,6 +40,8 @@ the specific language governing permissions and limitations under the License.
     <uses-permission android:name="android.permission.MANAGE_ACCOUNTS" />
     <!-- Get permission for reliable local dispatching on non-Google Play devices. -->
     <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.DISABLE_KEYGUARD"/>
```

Comment:
We should not be asking for additional permissions for release builds. I believe you can add these to `src/debug/AndroidManifest.xml`.